docs/reference/errors-and-warnings/NU1008.md

```diff
@@ -0,0 +1,42 @@
+---
+title: NuGet Error NU1008
+description: NU1008 error code
+author: nkolev92
+ms.author: nikolev
+ms.date: 11/07/2022
+ms.topic: reference
+f1_keywords: 
+  - "NU1008"
+---
+
+# NuGet Error NU1008
+
+<pre>Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: PackageId.</pre>
+
+### Issue
+
+When using central package management, versions must be defined on the PackageVersion item.
+
+In your project file, you may see:
+
+```xml
+<!-- In the project file. -->
+<PackageReference Include="PackageId" Version="5.1.0" />
+```
+
+### Solution
+
+- Remove the version from the PackageId PackageReference.
+- You may need to add or update the PackageVersion item for PackageId in Directory.Packages.props
+
+Example:
+
+```xml
+<!-- In the project file. -->
+<PackageReference Include="PackageId" />
+```
+
+```xml
+<!-- In the project file -->

```

Comment:
```suggestion
<!-- In the Directory.Packages.props file -->

```

Reply:
I saw it after I pushed, so I'm choosing not to give you credit for this.

Reply:
Do I at least get credit for reviewing your change within 120 seconds of when you pushed it?